lunavl/sdk/estimators/face_estimators/mask.py

```diff
@@ -56,16 +58,6 @@ class Mask(BaseEstimation):
         - mask
     """
 
-    #  pylint: disable=W0235
-    def __init__(self, mask: MedicalMaskEstimation):
```

Comment:
for what this change?

Reply:
this method is superfluous

Reply:
Base class takes any estimation. this class takes only MedicalMaskEstimation. this are reflected in the typing.

Reply:
FIxed